user_interactions/user_interaction_txt_text.py

```diff
@@ -0,0 +1,69 @@
+import os
+from datetime import datetime
+
+from user_interactions.base_user_interaction import (BaseUserInteraction,
+                                                     MessageScope)
+
+
+class UserInteractionFileManager():
+    def __init__(self, history_dir):
+        self.history_dir = history_dir
+        if not os.path.exists(self.history_dir):
+            os.mkdir(self.history_dir)
+
+        current_date = datetime.now().strftime('%Y-%m-%d-%H.%M.%S')
+        self.dir = os.path.join(self.history_dir, f"Game-{current_date}")
+        os.mkdir(self.dir)
+
+    def update_history_text(self, player_name, text):
```

Comment:
For discussion:
same for update_history_text name.
That not what method do. All the method do is _print to file message for particular player. That should be represented in method name